tests/e2e/specs/api/DevfileAcceptanceTestAPI.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { KubernetesCommandLineToolsExecutor } from '../../utils/KubernetesCommandLineToolsExecutor';
+import { DevWorkspaceConfigurationHelper } from '../../utils/DevWorkspaceConfigurationHelper';
+import { ShellString } from 'shelljs';
+import { expect } from 'chai';
+import { DevfileContext } from '@eclipse-che/che-devworkspace-generator/lib/api/devfile-context';
+import { TestConstants } from '../../constants/TestConstants';
+import { GitUtil } from '../../utils/vsc/GitUtil';
+
+suite(`Acceptance devfile registry test`, async function (): Promise<void> {
```

Comment:
There is different word order in test script name: `Devfile Acceptance Test` compare to suite name `Acceptance devfile **registry** test`.
And "devfile test" name looks differently to "devfile registry test".

It is not critical, of course, but would be more consistent to have the same test name reflected in test script file name and in suite name printed in test log.

Reply:
To be honest, I could not choose the name because suite can test alone devfile and should test all devfiles in some registry with UDIs:

> Should be used as a PR check for [UDI](https://github.com/devfile/developer-images/)
> The list of samples should be grabbed automatically from [the devfile registry](https://github.com/eclipse-che/che-devfile-registry/tree/main/devfiles) main branch
> The PR Check should be reusable for other repos such as the che devfile registry and cncf devfile registry

Reply:
I think _**devfile test**_ will be better